lib/manageiq/appliance_console/evm_server.rb

```diff
@@ -1,21 +1,30 @@
 module ManageIQ
   module ApplianceConsole
     class EvmServer
+      class NotRunnableError < RuntimeError; end
+
       class << self
         def running?
           service.running?
         end
 
         def start!(enable: false)
+          raise NotRunnableError, "Cannot start #{I18n.t("product.name")}: #{not_runnable_reason}" unless runnable?
+
           service.start(enable)
         end
 
         def start(enable: false)
           start!(:enable => enable)
         rescue AwesomeSpawn::CommandResultError => e
-          say e.result.output
-          say e.result.error
-          say ""
+          say(e.result.output)
```

Comment:
Were the parens added here for rubocop?  If so, I'd also be ok with adding these to the exclude list in manageiq-style: https://github.com/ManageIQ/manageiq-style/blob/master/styles/base.yml#L123 (perhaps in a new section for highligh / appliance console)

Reply:
Yes and no, we actually use `say("")` more often than we use `say ""`
```
$ grep -r --exclude-dir=spec/ 'say(' |wc -l
252
$ grep -r --exclude-dir=spec/ 'say ' | wc -l
32
```

So I was trying to be more consistent with the more common calling convention

Reply:
Sounds good :+1: